fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/InitializeLogicalResourceDenorms.java

```diff
@@ -115,8 +114,8 @@ private void runForDerby(IDatabaseTranslator translator, Connection c) {
                 }
                 updateStatement.setString(1, isDeleted);
                 updateStatement.setTimestamp(2, lastUpdated, SchemaConstants.UTC);
-                updateStatement.setLong(3, logicalResourceId);
-                updateStatement.setInt(4, versionId);
+                updateStatement.setInt(3, versionId);
+                updateStatement.setLong(4, logicalResourceId);
```

Comment:
this is a good catch but what was the result of the previous code?  I guess InitializeLogicalResourceDenorms.java was just broken for Derby?

Reply:
yes